Invoking command in '/home/microros/microros_ws/firmware/mcu_ws/uros/tracetools/tracetools_trace': PYTHONPATH=/home/microros/microros_ws/firmware/mcu_ws/install/tracetools_trace/lib/python3.8/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base /home/microros/microros_ws/firmware/mcu_ws/build/tracetools_trace build --build-base /home/microros/microros_ws/firmware/mcu_ws/build/tracetools_trace/build install --prefix /home/microros/microros_ws/firmware/mcu_ws/install/tracetools_trace --record /home/microros/microros_ws/firmware/mcu_ws/build/tracetools_trace/install.log --single-version-externally-managed
Invoked command in '/home/microros/microros_ws/firmware/mcu_ws/uros/tracetools/tracetools_trace' returned '0': PYTHONPATH=/home/microros/microros_ws/firmware/mcu_ws/install/tracetools_trace/lib/python3.8/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base /home/microros/microros_ws/firmware/mcu_ws/build/tracetools_trace build --build-base /home/microros/microros_ws/firmware/mcu_ws/build/tracetools_trace/build install --prefix /home/microros/microros_ws/firmware/mcu_ws/install/tracetools_trace --record /home/microros/microros_ws/firmware/mcu_ws/build/tracetools_trace/install.log --single-version-externally-managed
